src/components/Global/Sidebar/FavoritePools.tsx

```diff
@@ -35,14 +34,18 @@ export default function FavoritePools(props: propsIF) {
     );
 
     return (
-        <SidebarPoolsListContainer fontSize={'body'}>
-            <SidebarPoolsListHeaderContainer>
-                <SidebarPoolsListHeader>Pool</SidebarPoolsListHeader>
-                <SidebarPoolsListHeader>Volume</SidebarPoolsListHeader>
-                <SidebarPoolsListHeader>TVL</SidebarPoolsListHeader>
-            </SidebarPoolsListHeaderContainer>
+        <FlexContainer flexDirection='column' fontSize='body' fullHeight>
+            <HeaderGrid numCols={3} color='text2' padding='4px 0'>
+                {['Pool', 'Volume', 'TVL'].map((item) => (
```

Comment:
I like the idea - I think personal preference for readability in these situations would be just to have 3 divs listed out.  But it's totally just a preferences. 

10 Items for sure would map, 2 items for should would not.  The grey area is in between.